lucet-wasi/src/main.rs

```diff
@@ -173,13 +173,12 @@ fn main() {
         .value_of("timeout")
         .map(|t| Duration::from_millis(t.parse::<u64>().unwrap()));
 
-    let limits = Limits {
-        heap_memory_size,
-        heap_address_space_size,
-        stack_size,
-        globals_size: 0, // calculated from module
-        ..Limits::default()
-    };
+    let limits = Limits::default()
+        .with_heap_memory_size(heap_memory_size)
+        .with_heap_address_space_size(heap_address_space_size)
+        .with_stack_size(stack_size)
```

Comment:
If you `lucet-wasi` with a stack smaller than `32KiB` we'll panic here. I don't see a reason to make the hostcall reservation configurable here, so that seems fine?

Reply:
Done!